views/standard/index.html

```diff
@@ -5,14 +5,57 @@
       debug: <%= debug %>
    };
 
-   require(['pullManager'], function(_manager) {
+   require(['pullManager', 'jquery'], function(_manager, $) {
       App.debugStart = function debugStart() {
          App.debug = true;
          _manager.trigger();
       };
+
+      // Color switcher
+      $(document).ready(function() {
+         var cscookie = "";
+         document.cookie.split(';').forEach(function(currentValue) {
+            var elem = currentValue.split('=');
+            if (elem[0] == "pulldasher-colorscheme") {
+               cscookie = elem[1];
+            }
+         });
```

Comment:
During the hackathon, I couldn't find if any of our libraries did it automatically. I wasn't feeling so hot though, so I just put this together and went home. 

I'm not morally opposed to anyone fixing it if they find a cookie parser in one of the libraries. 
Jquery's is 3rd party though, like copperwall said, and we don't include that anywhere atm.